cvat-ui/src/utils/enviroment.ts

```diff
@@ -29,3 +29,7 @@ export function customWaViewHit(pageName?: string, queryString?: string, hashInf
         }
     }
 }
+
+export function isPublicInstance(): boolean {
```

Comment:
I don't think that it is a right solution. Probably HealthCheck endpoint can return extra information. Need to think how to implement that. Another solution is to have the message as a variable in settings. Please think and propose the best solution.